pkg/api/customization/kontainerdriver/store.go

```diff
@@ -45,9 +47,9 @@ func (s *store) Delete(apiContext *types.APIContext, schema *types.Schema, id st
 		return nil, nil
```

Comment:
Do you still need the lister if we swap to ID?

Reply:
Yes, since we now check if the driver is a builtin